pkg/storage/storage_tests/module_storage/storage_test.go

```diff
@@ -23,6 +24,10 @@ import (
 	"github.com/gomods/athens/pkg/storage/mongo"
 )
 
+var (
+	testConfigFile = filepath.Join("../../../../config.test.toml")
```

Comment:
@rchakra3 can you get rid of the `/` characters in this call? `Join` will replace them with the OS specific separator, which is nice for Windows. for example:

```go
filepath.Join("my", "file")
```

becomes `my/file` on unix and `my\file` on windows